app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -32,13 +31,20 @@
         android:digits="0123456789/- "
         android:enabled="@{!viewModel.isReadOnly}"
         android:focusable="true"
+        android:fontFamily="sans-serif"
         android:hint="@string/fractions_hint_text"
         android:inputType="text"
         android:longClickable="false"
         android:maxLength="200"
-        android:padding="8dp"
+        android:minHeight="48dp"
+        android:paddingLeft="16dp"
```

Comment:
I suggest adding paddingStart and paddingEnd to support RTL

Reply:
Done